docs/src/glossary.md

```diff
@@ -14,14 +14,14 @@ Examples for supported forms a Hamiltonian are the following, from the most gene
 <img src="../assets/controlhams.svg" width="80%"/>
 ```
 
-In Eq. (G1), ``Ĥ_0`` is the [Drift Term](@ref) (which may be zero) and each term under the sum over ``l`` is a [Control Term](@ref). Each control term is a Hamiltonian that depends on a set of [control functions](#Control-Function) (or simply "controls"). The controls are the functions directly tunable via optimal control.
+In Eq. (G1), ``Ĥ_0`` is the [Drift Term](@ref) (which may be zero) and each term under the sum over ``l`` is a [Control Term](@ref). Each control term is a Hamiltonian that depends on a set of [control functions](#Control-Function) (or simply "controls"). The controls are the functions directly tunable via optimal control.
```

Comment:
Did you change the non-breaking space here? If so, please don't. The space after "Eq." should always be non-breaking.

Reply:
It seems that the my local text editor added these non-breaking spaces automatically. I will fix it. Sorry for this inconvenience!